### DISCUSSION TITLE
Logging and multiple devices

### DISCUSSION THREAD

**Discussion by brentvatne** [expo/expo#18336] [Proposals and Ideas]
In the deprecated Expo CLI Web UI it was possible to split logs into separate panes, so you can easily differentiate between the logs from one OS/device and another.

![1_6FtqV41zNwI2jpX9441joA](https://user-images.githubusercontent.com/90494/180334687-713daf59-8aa3-4118-b9cb-290b3f9326b5.png)
 > [Image from the "Announcing Expo Dev Tools (beta)" blog post in 2018](https://blog.expo.dev/announcing-expo-dev-tools-beta-c252cbeccb36)

With the terminal UI, all logs appear in a single buffer and it's not possible to easily determine which log came from which device or operating system.

<img width="1112" alt="image" src="https://user-images.githubusercontent.com/90494/180335764-251d941e-617f-4303-8e48-51b9b36ce569.png">

> Each log message here comes from a different device!

Let's use this discussion thread to talk about how we might approach logging and multiple devices in the new local CLI.

**Comment by Codelica** [comment]
Yes... If nothing else, perhaps a command letter like the others (a, i, r, m, etc) to "f"ilter the log output to a specific device? there is already shift+a and shift+i to do similar for simulators.   In the end I do miss the convenient Web UI though :/

**Comment by luisnquin** [comment]
This!
